kubevela.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: kubevela
+  version: 1.7.0
+  epoch: 0
+  description: KubeVela is a modern application delivery platform that makes deploying and operating applications across today's hybrid, multi-cloud environments easier, faster and more reliable
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/kubevela/kubevela/archive/refs/tags/v${{package.version}}.tar.gz
+      expected-sha256: fd102cc6fd76273d5a3bc88faa050f3d0eed5cbfc9a253522d9fad012cc0babf
+  - runs: |
+      export CGO_ENABLED=0
+      export GO111MODULE=on
```

Comment:
do these exports persist to the next step?

Reply:
any ideas @tuananh?

Reply:
AFAIK, they don't ☝️